third_party/com_github_tuist_xcodeproj/BUILD

```diff
@@ -0,0 +1,7 @@
+# MARK: - Integration Test Related Targets
```

Comment:
```suggestion
# Integration test related targets
```

Can you apply the same re-formatting to all of these similar comments?

Reply:
Done.